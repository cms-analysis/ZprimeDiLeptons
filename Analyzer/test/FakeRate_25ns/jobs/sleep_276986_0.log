000 (276986.000.000) 08/30 15:32:06 Job submitted from host: <90.147.169.224:9618?addrs=90.147.169.224-9618&noUDP&sock=2441165_0900_3>
...
001 (276986.000.000) 08/30 15:46:35 Job executing on host: <90.147.168.210:41363>
...
006 (276986.000.000) 08/30 15:46:46 Image size of job updated: 45976
	44  -  MemoryUsage of job (MB)
	44836  -  ResidentSetSize of job (KB)
...
006 (276986.000.000) 08/30 15:51:46 Image size of job updated: 158932
	155  -  MemoryUsage of job (MB)
	158700  -  ResidentSetSize of job (KB)
...
005 (276986.000.000) 08/30 15:52:06 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:03:53, Sys 0 00:00:41  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:03:53, Sys 0 00:00:41  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	98247  -  Run Bytes Sent By Job
	69101  -  Run Bytes Received By Job
	98247  -  Total Bytes Sent By Job
	69101  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   198391       75   8767370
	   Memory (MB)          :      155     2500      2500
...
