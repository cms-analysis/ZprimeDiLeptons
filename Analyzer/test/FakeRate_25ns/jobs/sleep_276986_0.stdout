Running ZprimeMuMu Analysis_FR with executables RunZprimeMuMuAnalysis_FR
Working dir is /home/condor/dir_20740
Saving dir is /lustre/cms/store/user/defilip/ZprimeAnalysis/80X/histos/histosZprimeMuMu_FR
Compiling the macros
